### PR TITLE
JWT auth tutorial first draft

### PR DESCRIPTION
**Pull request by Srushtika**
@tomczoink @mattheworiordan this is the complete tutorial except for a few links to JWT docs, which I'll ensure after the docs are ready. Also, one of the images needs to be replaced to preserve consistency with the others. Review appreciated. The tutorial is hosted here: https://github.com/ably/tutorials/blob/jwt-authentication-node

### CODE REVIEW

**Comment by paddybyers** [comment]
Please don't commit the `.DS_Store` files

File: content/tutorials/jwt-authentication.textile [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+By definition, "JSON Web Tokens":https://jwt.io/ are an open, industry standard RFC 7519 method for representing claims securely between two parties. JWT auth is especially helpful in scenarios where users want to use a single authentication scheme for all of their users’ devices/browsers as well as other vendors and third party platforms such as Ably. JWT auth is also a convenient way to authenticate your IoT devices or servers which are built using the platforms that Ably doesn't yet support. This is possible because you don't have to instantiate an Ably client when using the JWT auth scheme. 
+
+Ably offers two different ways in which you can use a JSON Web Token to authenticate your users. You can either use the JWT as the token itself by formatting it correctly (called an Ably JWT), or pass to Ably a JWT of any structure which contains an Ably Token. We'll use the latter.
```
Comment (tomczoink) [inline content/tutorials/jwt-authentication.textile]: 'the latter' => 'the former' I believe.

Comment (Srushtika) [reply content/tutorials/jwt-authentication.textile]: Right, corrected. Thanks @tomczoink

**Comment by Srushtika** [comment]
@tomczoink  @mattheworiordan any final review before this is ready to be merged?

**Pull request merged**